lua/rsync/sync.lua

```diff
@@ -30,6 +31,8 @@ local function safe_sync(command, on_start, on_exit)
             if vim.inspect(output) ~= vim.inspect({ "" }) then
                 log.info(string.format("safe_sync command: '%s', on_stderr: '%s'", command, vim.inspect(output)))
             end
+
+            config.values.on_error(output, command)
         end,
 
```

Comment:
```suggestion
        end,
        config.values.on_stderr(output, command)
```

Reply:
Sry I think I don't understand you this then. Why would I put it out of the hook? This is a table so this won't work then?

Reply:
Me neither, must have seen wrong. 😄